tests/integration/interactive_dev_test.go

```diff
@@ -93,7 +93,42 @@ var _ = Describe("odo dev interactive command tests", func() {
 					helper.ExpectString(ctx, "Enter component name")
 					helper.SendLine(ctx, "my-app")
 
-					helper.ExpectString(ctx, "Press Ctrl+c to exit")
+					helper.ExpectString(ctx, "[Ctrl+c]")
+					ctx.StopCommand()
+				})
+
+			Expect(helper.ListFilesInDir(commonVar.Context)).To(ContainElements("devfile.yaml"))
+		})
+
+		It("should run alizer and then be able to run manual syncing", func() {
+
+			language := "python"
+			_, _ = helper.RunInteractive([]string{"odo", "dev", "--random-ports"},
```

Comment:
As I commented out in https://github.com/redhat-developer/odo/pull/6045#pullrequestreview-1087171779, would it  be possible to add a test (or enrich the existing one) that makes use of the "--no-watch" flag (see [cmd_dev_test.go](https://github.com/redhat-developer/odo/blob/main/tests/integration/cmd_dev_test.go#L369-L394))?

The [existing test](https://github.com/redhat-developer/odo/blob/main/tests/integration/cmd_dev_test.go#L369-L394) currently makes sure that no push is triggered after changing a file while running `odo dev --no-watch`. Per the acceptance criteria, I think we could enrich it to make sure that updating a file and pressing the "p" key triggers a sync.. Same point without the "--no-watch" flag.
To me, this behavior is important to test to ensure no future regression.